tools/scripts/xabuild

```diff
@@ -160,6 +160,14 @@ function ConfigureLocalXbuild()
 	done
 }
 
+if [ -z "$CONFIGURATION" ]; then
+		for p in "$*"; do
+				case $p in
+						/p:Configuration=*|-p:Configuration=*) CONFIGURATION="`echo $p | cut -d '=' -f 2`" ;;
```

Comment:
One problem with this is that people might not use Debug or Release configurations. For example, they could add a new `Android-Debug` configuration (e.g. if the solution has iOS projects as well, and they don't want to build iOS stuff when doing Android stuff), in which case `CONFIGURATION` would be set to `Android-Debug`, which would fail "weirdly" elsewhere.

If we move this logic to be before line 44, when `$prefix` and `$Paths_targets` are set, then it should be fine (-ish?), as the `[ -d "$prefix/../../bin/$c" ]` check on line 50 would prevent use of "invalid" `CONFIGURATION` values.